cli/cmd/cli_state.go

```diff
@@ -183,6 +183,7 @@ func (c *cliState) NewClient() error {
 	client, err := api.NewClient(c.Account,
 		api.WithLogLevel(c.LogLevel),
 		api.WithApiKeys(c.KeyID, c.Secret),
+		api.WithTimeout(time.Second*125),
```

Comment:
We purposely set the timeout to be 5 seconds more than the one configured in NGINX so that we let it close the connection instead of the CLI. That will make our monitoring system display failures/error-codes.